### ISSUE TITLE
iOS Support

### ISSUE THREAD

**Issue by ekimlinger** [rafaellincoln/react-native-telephony#9]
Are there going to be any plans to implement this in iOS?

**Comment by ekimlinger** [comment]
Looks like we might be able to use [the ios core telephony framework](https://developer.apple.com/documentation/coretelephony?changes=_3) in order to get some, if not most, of the same information.

**Comment by psyzz** [comment]
@ekimlinger did you successfully implement this for iOS ?

**Comment by ekimlinger** [comment]
Nope, wasn't critical to business needs, unfortunately, so we skipped it.

**Comment by psyzz** [comment]
Ok thanks @ekimlinger , i think some data on iOS can only get retrieve with Private API
